### ISSUE TITLE
Automatic inverse text color

### ISSUE THREAD

**Issue by purplespektr** [ramensoftware/windows-11-notification-center-styling-guide#8]
Please add the feature for when there's something white behind the notification center

**Comment by adaehri** [comment]
Yes, please. Everything is almost invisible when there's something white underneath 
![image](https://github.com/user-attachments/assets/e16b5f03-63f5-41a9-b521-8780c1a82931)
